modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -937,7 +937,7 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     return false;
   };
 
-  const isEmpty = (node: Node, elements?: Record<string, any>) => {
+  const isEmpty = (node: Node, elements?: Record<string, any>, includeZwsp?: boolean) => {
```

Comment:
ok, I agree that add new param is not the best solution, but I did not get how to implement `isWhitespace`, should I create it as a new function that use `isEmpty` and add the check for `zwsp`?

Reply:
ok @spocke , but passing a predicate instead of the flag could be almost the same thing, since is not the only check that we have in it.

could it be better to instead of add a flag or a predicate to add an object and have a signature like this?
```
const isEmpty = (node: Node, elements?: Record<string, any>, data?: ({ includeZwsp?: boolean }))
```

Reply:
ok, I changed with the object, let me know if it is ok